backend/inspirehep/search/facets.py

```diff
@@ -28,6 +28,30 @@ def inner(values):
     return inner
 
 
+def date_range_contains_conferences_filter():
+    def inner(values):
+        opening_date, closing_date = values[0].split("--")
```

Comment:
don't like that tbh, what will happen with ``contains=`` (empty)?

Reply:
It's for a single use case for now, that's why I didn't care to handle all the cases. It should blow up pretty obviously, so if it happens we will see and handle the case.